_posts/2019-12-16-event-store-6.0.0-preview-release.md

```diff
@@ -0,0 +1,380 @@
+---
+title: "Event Store 6.0.0 Preview 1"
+author: "Hayley Campbell"
+layout: blog-post
+---
+
+We are very excited to announce the first preview release of Event Store 6.0.0!  
+With this release we want to show you where we are planning on taking Event Store in the future.
+
+This release is not intended to be used in production and is still rough around the edges, however we wanted to provide you with the release bits and we welcome your feedback.
+
+If you encounter any issues, please don’t hesitate to open an issue on GitHub if there isn’t one already.
+
+## Important!
+
+With the preview release of Event Store, Event Store will only expose the external HTTP interface over HTTPS.
+
+This requires an TLS certificate, but for ease of use we have introduced a development mode which will use a self signed certificate intended for development use only.
+
+# What’s new in Event Store
+
+There are a number of exciting features included in this preview:
+- The move to .NET Core 3.1
+- A gRPC client
+- Server Side Filtering
+- Read-Only Replicas
+- Master Resignation
+- Restarting the Projection Subsystem
+- Pinned by Correlation Id strategy for Competing Consumers
+
+Given that this is a preview release, there is no documentation on the new features at the moment.
+
+We have included details on how to get started with each feature further along in this blog post.  
+Additionally, you can reach out to us over github or google groups to ask questions or report any issues.
```

Comment:
```suggestion
Additionally, you can reach out to us over [github](https://github.com/EventStore/EventStore) or [google groups](https://groups.google.com/forum/#!forum/event-store) to ask questions or report any issues.
```

Reply:
I would rather people approached us via github issues only really. otherwise we split conversations...